### PR TITLE
Added mealie container template

### PR DESCRIPTION
**Pull request by nickcj931**
closes https://github.com/hay-kot/mealie/issues/37

### CODE REVIEW

**Comment by jrdnlc** [comment]
Do you have an icon we can add? Also, change the branch to `testing`

**Comment by nickcj931** [comment]
Sorry no icon at present. I will add this in once that is in place.

**Comment by nickcj931** [comment]
Noticed an issue when running this template yesterday, on first launch it throws an error. When you relaunch it works fine. This is fixed in the next push. However in the interest of saving some one a headache i will close this pull request and re-open when the new image is live.

File: templates/mealie.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  <WebUI>http://[IP]:[PORT:9000]</WebUI>
+  <DonateText>If you are not a coder, you can still contribute financially. financial contributions help me prioritize working on this project over others and helps me know that there is a real demand for project development.</DonateText>
+  <DonateLink>https://www.buymeacoffee.com/haykot</DonateLink>
+  <Description>Mealie is a self hosted recipe manager and meal planner with a RestAPI backend and a reactive frontend application built in Vue for a pleasant user experience for the whole family. Easily add recipes into your database by providing the url and mealie will automatically import the relevant data or add a family recipe with the UI editor.</Description>
+  <Config Name="db_type" Target="db_type" Default="sqlite" Mode="" Description="The database type to be used. Current Options 'sqlite' and 'mongo'" Type="Variable" Display="always" Required="true" Mask="false">sqlite</Config>
+  <Config Name="Time Zone" Target="TZ" Default="America/Anchorage" Mode="" Description="You should set your time zone accordingly so the date/time features work correctly" Type="Variable" Display="always" Required="true" Mask="false"/>
```
Comment (FoxxMD) [inline templates/mealie.xml:17]: This is not necessary, unraid automatically adds this env to the run command

**Pull request closed without merging**